introduction.md

```diff
@@ -110,16 +110,16 @@ The audit is presented as a review of the contribution. The codebase steward giv
 
 For the codebase to be completely certified every meaningful line of code, and the commits behind the code, need to meet the Standard.
 
-If codebases have been completely audited from the first merge request they can be completely be certified as compliant with the Standard for Public Code immediately.
+If codebases have been completely audited from the first merge request they can be immediately certified as compliant with the Standard for Public Code.
 
-If the audit process is added to an existing codebase, the new merge requests can be certified, but the existing code cannot be certified. By auditing every new merge request the codebase can move towards being completely certified.
+If the audit process is added to an existing codebase, the new merge requests can be certified, but the existing code cannot be certified. By auditing every new merge request the codebase can move incrementally towards being completely certified.
 
 ## The goals for the Standard for Public Code
 
 This Standard supports developers, designers, business management and policy makers to:
 
 * develop high quality software and policy for better public service delivery
-* develop reusable codebases that can be reused across contexts and collaboratively maintained
+* develop buildable codebases that can be reused across contexts and collaboratively maintained
```

Comment:
```suggestion
* develop codebases that can be reused across contexts and collaboratively maintained
```

I approve of the deduping of 'reusable', but 'buildable' doesn't seem directly relevant. Do you mean something closer to 'hackable' (as opposed to a monolithic black box)?

Reply:
Thanks for considering my suggestions. I concur with your suggested change.

It's not really a proper English word yet, but "buildability" is a different thing than reusability and maybe more powerful in the context of the public code standard you are promoting. Reusable connotes more a process of forking the code and using it in a completely different context, which is also cool. Buildable to me is more about either building onto a codebase in its original context, making it better, or using that codebase directly to build some other capability. It also relates to being able to complete a build process, technically in a CI/CD sense. In the same bullet, we have "collaboratively maintained," which is hugely important. But I'd also say maintenance implies potentially just keeping up with bug fixes and dependency changes, while if the core concept is buildable, that suggests that future collaboration might be adding features and capabilities.

Hackable could work in some respects but seems to connote a less formal process than buildable. It is also a good thing, though, because it implies a level of rapid accessibility and ease of experimentation, which I think we also want to promote.

All three of these are important concepts with slightly different semantics that are already somewhat addressed elsewhere in the criteria. For the intro, your suggested change makes good sense without confusing the situation with all this other stuff. I'll ponder a bit more on the semantics of reusable, buildable, and hackable and see if I come up with any other suggestions for elsewhere.